index.html

```diff
@@ -100,7 +95,108 @@ <h6>Skills</h6>
         <p class="left">MY RECENT WORK</p>
     </div>
     <hr>
-    
-    
+    <section class="works">
+        <img src="./ImgPlaceholder.png" alt="">
+        <h2 class="post-heading">Multi-Post Stories</h2>
+        <p class="post-1">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <section class="language">
+            <h5>css</h5>         
+                <img class="line" src="./Rectangle70.png" alt="">
+            <h5>html</h5>
+                <img class="line" src="./Rectangle70.png" alt="">
+            <h5>bootstrap</h5>
+            <img class="line" src="./Rectangle70.png" alt="">
+                <h5>ruby</h5>        
+        </section>
+        <button class="btn" type="button">See Project</button>
+        <img src="./ImgPlaceholder.png" alt="">
+        <h2 class="post-heading">Multi-Post Stories</h2>
+        <p class="post-1">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <section class="language">
+            <h5>css</h5>         
+                <img class="line" src="./Rectangle70.png" alt="">
+            <h5>html</h5>
+                <img class="line" src="./Rectangle70.png" alt="">
+            <h5>bootstrap</h5>
+            <img class="line" src="./Rectangle70.png" alt="">
+                <h5>ruby</h5>        
+        </section>
+        <button class="btn" type="button">See Project</button>
```

Comment:
- [ ] I can see that the structure you have used to represent a single project isn't the right one, and could break your code in the future while working on the desktop version of your portfolio, let's try fixing this by doing the following:
- Let's create a single container `<div>` to wrap all the content of the project details including the image, name, languages, and the `see Project` button.
- After that, you can use a `<ul>` and `<li>` for the languages, and remember you don't need to use an image for the border-right after each language, see the screenshot below 👇 
![image](https://user-images.githubusercontent.com/55465133/133835042-9dad4b9d-9349-4d10-8d55-54308a6cb4c0.png)
you can accomplish this by using the following technique:

HTML File
```
<ul class="project-languages">
   <li>CSS</li>
   <li>HTML</li>
   <li>Bootstrap</li>
</ul>
```
CSS file
```
.project-languages > li:not(:last-child){
    border-right: 1px solid #555555
}
```
This will ensure the border-left is not available in the last language as what we have from the screenshot above ☝️.

- [ ] You can also add the button interactions on all the buttons such as hover, active as shown in the project requirement  👇 
![image](https://user-images.githubusercontent.com/55465133/133835835-c852fc55-b6dd-4d35-944b-8905857f4a7c.png)
![image](https://user-images.githubusercontent.com/55465133/133835879-7348b6d5-edb9-45e7-948e-824bb93e96ca.png)

- [ ] Kindly remember to have all the projects in one single container that can also be a `<div>` tag, and should have the `display: grid;` to align all the projects. Remember you are the best, I am sure you can do this 😉